.github/workflows/docker-build.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           else
             export FF_BUILD_ALL_INFERENCE_EXAMPLES=OFF
           fi
-          ./docker/build.sh flexflow
+          ./docker/build.sh --image_name flexflow --cuda_version 11.8
```

Comment:
let's see if this works, and then we can switch to building docker images for several versions of CUDA in CI!